tfs/src/main/java/hudson/plugins/tfs/model/AbstractHookEvent.java

```diff
@@ -112,7 +116,27 @@ static JSONObject fromResponseContributors(final List<GitStatus.ResponseContribu
                     return new TeamEventsEndpoint.ScheduledResponseContributor(project);
                 }
 
-                final TeamPushTrigger pushTrigger = TeamEventsEndpoint.findTrigger(job, TeamPushTrigger.class);
+                boolean shouldRun = false;
+                FlowDefinition jobDef = ((WorkflowJob)job).getDefinition();
+                if (jobDef instanceof CpsScmFlowDefinition)
+                {
+                    GitSCM jobSCM = (GitSCM)((CpsScmFlowDefinition) jobDef).getScm();

```

Comment:
jobSCM should be final
Are you sure that the SCM provider is Git here? we support TFVC as well. You should probably check the type before you cast it.